### PR TITLE
Feature/continuation of home page

### PR DESCRIPTION
**Pull request by MoneyManAnt06**
Check everything EXCEPT( section-update and section gaming)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -84,7 +84,254 @@ <h3 class="Upcoming-games-title">MARVEL SNAP</h3>
       </div>
     </div>
   </section>
-  <section class="subscribe-container">
+
+  <section class="Section-update">
```
Comment (MoneyManAnt06) [inline index.html]: dont check section

File: index.html [hunk 2]
```diff
@@ -84,7 +84,254 @@ <h3 class="Upcoming-games-title">MARVEL SNAP</h3>
...
+        </div>
+        <div class="Gallery-image-right">
+          <div>
+            <img src="/public/avengers-portrait.jpeg" alt="">
+          </div> 
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="Section-gaming">
```
Comment (MoneyManAnt06) [inline index.html]: dont check section

File: index.html
Comment (robSamuel) [inline index.html:1]: Please use the header that Harrisong created

File: index.html [hunk 3]
```diff
@@ -84,7 +84,254 @@ <h3 class="Upcoming-games-title">MARVEL SNAP</h3>
...
+    </div>
+  </section>
+
+  <section class="Section-gallery">
+    <div class="Gallery-container">
+      <h2>Marvel Gallery</h2>
+      <div class="Gallery-image-split">
+        <div class="Gallery-image-left">
+          <div class="Gallery-left-pics">
+            <img src="/public/helicarrier.jpeg" alt="">
```
Comment (robSamuel) [inline index.html]: Every **img** tag should have the alt description

File: index.html [hunk 4]
```diff
@@ -84,7 +84,254 @@ <h3 class="Upcoming-games-title">MARVEL SNAP</h3>
...
+        </p>
+      </div>
+
+    </div>
+  </section>
+
+  <section class="Section-pricing">
+    <div class="Pricing-container">
+      <div class="Pricing-title">
+        <h1>Pricing Plans</h1>
```
Comment (robSamuel) [inline index.html]: This tag should be an **h2**

File: index.html [hunk 5]
```diff
@@ -84,7 +84,254 @@ <h3 class="Upcoming-games-title">MARVEL SNAP</h3>
...
+      </div>
+      <div class="Pricing-packages">
+        <div class="Pricing-padding">
+          <div class="Pricing-background">
+            <h3 class="Pricing-package-title">Silver Package</h3>
```
Comment (robSamuel) [inline index.html]: Change the **h3** and **h1** tags

File: index.html [hunk 6]
```diff
@@ -84,7 +84,254 @@ <h3 class="Upcoming-games-title">MARVEL SNAP</h3>
...
+            <div class="Pricing-button">
+              <a class="Pricing-tag" href="#">Buy Now </a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="Section-questions">
```
Comment (robSamuel) [inline index.html]: Rename this to **Questions**

File: index.html [hunk 7]
```diff
@@ -84,7 +84,254 @@ <h3 class="Upcoming-games-title">MARVEL SNAP</h3>
...
+          </div>
+        </div>
+      </div>
+  </div>
+  </section>
+
+  <section class="Section-latest-blog">
+    <div class="Pricing-container">
+      <div class="Pricing-title">
+        <h1>Latest Blog Posts</h1>
```
Comment (robSamuel) [inline index.html]: The page should have only one h1

File: src/scss/pages/_homePage.scss [hunk 8]
```diff
@@ -0,0 +1,337 @@
...
+  &-gallery {
+    padding: 120px 0 90px;
+  }
+
+  &-gaming {
+    padding: 120px 0;
+    background-image: url(/public/playstation-cover.png);
+    background-size: cover;
+
+    // padding: 120px 0;
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Delete all of these comments

File: src/scss/pages/_homePage.scss [hunk 9]
```diff
@@ -0,0 +1,337 @@
...
+    background-color: $home-darkgrey;
+  }
+
+  &-text {
+    width: 50%;
+    color: $home-white;
+  }
+
+  &-image {
+    width: 50%;
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Delete this and use the classes that bootstrap provides.

File: src/scss/pages/_homePage.scss [hunk 10]
```diff
@@ -0,0 +1,337 @@
...
+  }
+}
+
+.Upcoming {
+  &-games-container {
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Delete this and use the classes that bootstrap provides.

File: src/scss/pages/_homePage.scss [hunk 11]
```diff
@@ -0,0 +1,337 @@
...
+    height: 339px;
+    padding: 15px;
+
+    &-image {
+      border-radius: 200px;
+      width: 100%;
+    }
+
+    &-title {
+      text-align: center;
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Delete this and use the classes that bootstrap provides.

File: src/scss/pages/_homePage.scss [hunk 12]
```diff
@@ -0,0 +1,337 @@
...
+    }
+  }
+}
+
+.Update {
+  &-mask {
+    width: 100%;
+    height: 100%;
+    background-color: rgba(154, 0, 154, 0.4);
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Add this value to the variables

File: src/scss/pages/_homePage.scss [hunk 13]
```diff
@@ -0,0 +1,337 @@
...
+  &-list {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+
+    :hover {
+      background-color: aqua;
+    }
+
+    &-container {
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Delete this and use the classes that bootstrap provides.

File: src/scss/pages/_homePage.scss [hunk 14]
```diff
@@ -0,0 +1,337 @@
...
+      display: flex;
+      flex-direction: row;
+    }
+  }
+
+  &-image {
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Delete this and use the classes that bootstrap provides.

File: src/scss/pages/_homePage.scss [hunk 15]
```diff
@@ -0,0 +1,337 @@
...
+    height: 100%;
+    width: 100%;
+
+    &-container {
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Delete this and use the classes that bootstrap provides.

File: src/scss/pages/_homePage.scss [hunk 16]
```diff
@@ -0,0 +1,337 @@
...
+      width: 50%;
+    }
+  }
+
+  &-text {
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Delete this and use the classes that bootstrap provides.

File: src/scss/pages/_homePage.scss [hunk 17]
```diff
@@ -0,0 +1,337 @@
...
+    }
+
+    &-right {
+      width: 20%;
+      height: 100%;
+      padding: 0 15px;
+    }
+  }
+
+  &-left-pics {
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Delete this and use the classes that bootstrap provides.

File: src/scss/pages/_homePage.scss [hunk 18]
```diff
@@ -0,0 +1,337 @@
...
+    padding: 0 15px;
+  }
+
+  &-packages {
+    display: flex;
+    justify-content: center;
+  }
+
+  &-background {
+    color: #f2f2f2;
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Use variables for the colors

File: src/scss/pages/_homePage.scss [hunk 19]
```diff
@@ -0,0 +1,337 @@
...
+    text-align: center;
+    margin: 0;
+    padding: 35px 30px 25px;
+  }
+
+  &-price {
+    font-size: 36px;
+    padding: 30px;
+    text-align: center;
+    background: #342b7e;
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Use variables for the colors

File: src/scss/pages/_homePage.scss [hunk 20]
```diff
@@ -0,0 +1,337 @@
...
+    width: 100%;
+    padding-bottom: 50px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  &-tag {
+    text-decoration: none;
+    border: 2px solid;
+    border-image: linear-gradient(to right, #3fcaff 0%, #a4ffb0 51%, #3fcaff 100%);
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Use variables for the colors

File: src/scss/pages/_homePage.scss [hunk 21]
```diff
@@ -0,0 +1,337 @@
...
+    border-image-slice: 1;
+    color: #fff;
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Use variables for the colors

File: src/scss/pages/_homePage.scss [hunk 22]
```diff
@@ -0,0 +1,337 @@
...
+    display: inline-block;
+    overflow: hidden;
+    padding: 0 60px;
+    line-height: 46px;
+
+    &:hover {
+      background: linear-gradient(to right, #3fcaff 0%, #a4ffb0 51%, #3fcaff 100%);
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Use variables for the colors

File: src/scss/pages/_homePage.scss [hunk 23]
```diff
@@ -0,0 +1,337 @@
...
+      color: black;
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Use hexadecimals values for the colors

File: src/scss/pages/_homePage.scss [hunk 24]
```diff
@@ -0,0 +1,337 @@
...
+  }
+
+  &-main-title {
+    text-align: center;
+    margin-bottom: 60px;
+  }
+
+  &-container {
+    display: flex;
+    border-bottom: 2px dotted grey;
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Use variables for the colors

File: src/scss/pages/_homePage.scss [hunk 25]
```diff
@@ -0,0 +1,337 @@
...
+    margin: 0;
+    padding-top: 40px;
+  }
+
+  &-text {
+    padding: 30px 20px 40px 0;
+  }
+
+  &-last-border {
+    border-bottom: none;
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Please check if bootstrap have a class for this

File: src/scss/pages/_homePage.scss [hunk 26]
```diff
@@ -0,0 +1,337 @@
...
+  }
+}
+
+.Latest-blog {
+  &-image {
+    width: 100%;
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: USe the bootstrap classes

**Review by robSamuel — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (MoneyManAnt06) [reply index.html]: 👍🏼

File: index.html (hunk 4 above)
Comment (MoneyManAnt06) [reply index.html]: 👍🏼

File: index.html (hunk 6 above)
Comment (MoneyManAnt06) [reply index.html]: 👍🏼

File: src/scss/pages/_homePage.scss (hunk 25 above)
Comment (MoneyManAnt06) [reply src/scss/pages/_homePage.scss]: 👍🏼

File: index.html
Comment (MoneyManAnt06) [reply index.html:1]: 👍🏼

File: index.html (hunk 5 above)
Comment (MoneyManAnt06) [reply index.html]: 👍🏼

File: src/scss/pages/_homePage.scss (hunk 26 above)
Comment (MoneyManAnt06) [reply src/scss/pages/_homePage.scss]: 👍🏼

File: index.html [hunk 27]
```diff
@@ -1,96 +1,336 @@
...
-</head>
-<body>
-  <div class="container">
-    <div class="NavBar">
-      <div class="NavBar-Image">
-        <img class="Marvel-Image" src="/Public/marvel.png" alt="Marvel Logo" />
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Game Page</title>
```
Comment (robSamuel) [inline index.html]: Why the title of this page is **Game Page** instead of something related with marvel?

File: index.html [hunk 28]
```diff
@@ -1,96 +1,336 @@
...
+                inappropriate behavior is often laughed off as “boys will be boys,” women face higher conduct standards
+                especially in the workplace. That's why it's crucial that as women.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="Section-latest-blog">
```
Comment (robSamuel) [inline index.html]: Remove the word **Section** from the class name

File: index.html [hunk 29]
```diff
@@ -1,96 +1,336 @@
...
+          <h2 class="Common-main-title">Latest Blog Posts</h2>
         </div>
-        <div class="Upcoming-games">
-          <div>
-            <img class="Upcoming-games-image" src="/public/marvelsnap-cover.png" alt="">
-            <h3 class="Upcoming-games-title">MARVEL SNAP</h3>
+        <div class="d-flex justify-content-center">
+          <div class="Pricing-padding">
+            <div class="Pricing-background">
+              <div class="Latest-blog-image-container">
```
Comment (robSamuel) [inline index.html]: The nesting of this class has 4 levels.

File: index.html [hunk 30]
```diff
@@ -1,96 +1,336 @@
...
+                  <a href=""><i class="fa-regular fa-calendar-days"></i> 23rd Nov</a>
+                  <a href=""><i class="fa-solid fa-heart"></i>16</a>
+                  <a href=""><i class="fa-solid fa-comments"></i>06</a>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="Pricing-padding">
+            <div class="Pricing-background">
+              <div class="Latest-blog-image-container">
```
Comment (robSamuel) [inline index.html]: The nesting of this class has 4 levels.

File: index.html [hunk 31]
```diff
@@ -1,96 +1,336 @@
...
```
Comment (robSamuel) [inline index.html]: The nesting of this class has 4 levels.

File: src/pages/gallery.html [hunk 32]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (robSamuel) [inline src/pages/gallery.html]: The title of this page should be related with Gallery

File: src/scss/pages/_contact.scss [hunk 33]
```diff
@@ -1,130 +1,250 @@
...
+      color: $black-color;
+    }
+
+    &-inputs {
+      height: 214px;
+      width: 853px;
+    }
+
+    &-form {
+      &-box {
```
Comment (robSamuel) [inline src/scss/pages/_contact.scss:167]: Remove this class, there's no need of it, you can use the bootstrap class just for this.

File: src/scss/pages/_homePage.scss [hunk 34]
```diff
@@ -0,0 +1,125 @@
+.Section {
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Remove the word **Section** for the classes

File: src/scss/pages/_homePage.scss [hunk 35]
```diff
@@ -0,0 +1,125 @@
...
+    background-image: url(/public/playstation-cover.png);
+    background-size: cover;
+  }
+
+  &-latest-blog {
+    padding: 80px 0 120px;
+  }
+}
+
+.Gallery {
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Per each root class you should create a file.

File: src/scss/pages/_homePage.scss [hunk 36]
```diff
@@ -0,0 +1,125 @@
...
+    height: 270px;
+  }
+
+  &-review {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    &-name {
+      color: white;
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Do not use the name of the colors

File: src/scss/pages/_homePage.scss [hunk 37]
```diff
@@ -0,0 +1,125 @@
...
+    }
+
+    &-text {
+      color: white;
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Do not use the name of the colors

File: src/scss/pages/_homePage.scss [hunk 38]
```diff
@@ -0,0 +1,125 @@
...
+      text-align: center;
+      max-width: 480px;
+    }
+  }
+
+  &-rating {
+    padding: 0;
+    color: aqua;
```
Comment (robSamuel) [inline src/scss/pages/_homePage.scss]: Do not use the name of the colors

**Review by robSamuel — CHANGES_REQUESTED** [review]

File: src/scss/pages/_homePage.scss (hunk 26 above)
Comment (robSamuel) [reply src/scss/pages/_homePage.scss]: Delete .Latest-blog-image

File: src/scss/pages/_homePage.scss (hunk 22 above)
Comment (MoneyManAnt06) [reply src/scss/pages/_homePage.scss]: 👍🏼

File: src/scss/pages/_homePage.scss (hunk 23 above)
Comment (MoneyManAnt06) [reply src/scss/pages/_homePage.scss]: 👍🏼

File: src/scss/pages/_homePage.scss (hunk 24 above)
Comment (MoneyManAnt06) [reply src/scss/pages/_homePage.scss]: 👍🏼

File: index.html (hunk 27 above)
Comment (MoneyManAnt06) [reply index.html]: 👍🏼

File: index.html (hunk 28 above)
Comment (MoneyManAnt06) [reply index.html]: 👍🏼

File: index.html (hunk 29 above)
Comment (MoneyManAnt06) [reply index.html]: 👍🏼

File: index.html (hunk 30 above)
Comment (MoneyManAnt06) [reply index.html]: 👍🏼

File: index.html (hunk 31 above)
Comment (MoneyManAnt06) [reply index.html]: 👍🏼

File: src/scss/pages/_homePage.scss (hunk 34 above)
Comment (MoneyManAnt06) [reply src/scss/pages/_homePage.scss]: 👍🏼

File: src/scss/pages/_homePage.scss (hunk 35 above)
Comment (MoneyManAnt06) [reply src/scss/pages/_homePage.scss]: 👍🏼

File: src/scss/pages/_homePage.scss (hunk 36 above)
Comment (MoneyManAnt06) [reply src/scss/pages/_homePage.scss]: 👍🏼

File: src/scss/pages/_homePage.scss (hunk 37 above)
Comment (MoneyManAnt06) [reply src/scss/pages/_homePage.scss]: 👍🏼

File: src/scss/pages/_homePage.scss (hunk 38 above)
Comment (MoneyManAnt06) [reply src/scss/pages/_homePage.scss]: 👍🏼

**Review by robSamuel — APPROVED** [review]

**Review by robSamuel — APPROVED** [review]

**Pull request merged**